From 05ae78ec7ed867948259f400ba9640542f974c72 Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@debian.org>
Date: Sun, 18 Sep 2022 18:26:34 +0200
Subject: [PATCH 5/6] copyright-format: Use the URL format instead of the
 postal code for the GPL

While the GPL-2 recommends using a postal address in the notice to
include in source files, this has already changed once, and the GPL-3
already uses this new more future-proof format where the license is
referenced via an URL on the web.
---
 copyright-format-1.0.xml | 15 ++++++---------
 1 file changed, 6 insertions(+), 9 deletions(-)

diff --git a/copyright-format-1.0.xml b/copyright-format-1.0.xml
index cc7494c..8dece57 100644
--- a/copyright-format-1.0.xml
+++ b/copyright-format-1.0.xml
@@ -1236,9 +1236,8 @@ also delete it here.</programlisting>
  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  GNU General Public License for more details.
  .
- You should have received a copy of the GNU General Public License along
- with this package; if not, write to the Free Software Foundation, Inc.,
- 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ You should have received a copy of the GNU General Public License
+ along with this package; if not, see &lt;https://www.gnu.org/licenses/&gt;.
  .
  On Debian systems, the full text of the GNU General Public License
  version 2 can be found in the file '/usr/share/common-licenses/GPL-2'.</programlisting>
@@ -1291,9 +1290,8 @@ License: GPL-2+
  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  GNU General Public License for more details.
  .
- You should have received a copy of the GNU General Public License along
- with this package; if not, write to the Free Software Foundation, Inc.,
- 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ You should have received a copy of the GNU General Public License
+ along with this package; if not, see &lt;https://www.gnu.org/licenses/&gt;.
  .
  On Debian systems, the full text of the GNU General Public License
  version 2 can be found in the file '/usr/share/common-licenses/GPL-2'.
@@ -1379,9 +1377,8 @@ License: GPL-2+
  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  GNU General Public License for more details.
  .
- You should have received a copy of the GNU General Public License along
- with this package; if not, write to the Free Software Foundation, Inc.,
- 51 Franklin St, Fifth Floor, Boston, MA  02110-1301 USA.
+ You should have received a copy of the GNU General Public License
+ along with this package; if not, see &lt;https://www.gnu.org/licenses/&gt;.
  .
  On Debian systems, the full text of the GNU General Public License
  version 2 can be found in the file '/usr/share/common-licenses/GPL-2'.]]></programlisting>
-- 
2.37.2

